DataManagement/GitTutorial.Rmd

```diff
@@ -297,11 +357,83 @@ data/
 <br/><br/>
 <br/><br/>
 
+
 ## Useful terminal comands
-!!!Add a list of the most useful commands!!!
 
-### need to add this somewhere
-usethis::usethis::use_git_config()
+Most commands can also be executed in the terminal in RStudio (**Terminal** tab).
+Here is a list with a few imporant commands that might come in handy.
+
+### Clone a repo from GitHub
+
+To **clone** or make a local copy of a **remote** repository on GitHub type:
+
+```git clone https://github.com/git:kingsleyshacklebolt/dragon_study.git```
+
+
+### Check configurations
+
+To check the configurations of your repo type:
+
+```git remote -v```
+
+This command will show you the origin or your repo, basically from were you push and pull.
+If you have forked a repo it will also show you the upstream repo (see the tutorial on branches for more details).
+
+
+You might want to configure the information of your local repositories.
+This command sets the name you want attached to your commit transactions.
+Note that this will change the name for all your local repos.
+
+```
+$ git config --global user.name "yourname"
+$ git config --global user.name "kingsleyshaklebolt"
+```
+
+Similarly, you can sets the email you want attached to your commit transactions.
+
+```
+$ git config --global user.email "youremail"
+$ git config --global user.email "kingsleyshaklebolt@ministryofmagic.com"
+```
+
+You can also manually set the **remote** repository for your **local** repository.
+The url points to a repository on GitHub.
+
+```
+$ git remote add origin [url]
+$ git remote add origin https://github.com/git:kingsleyshacklebolt/dragon_study.git
+```
+
+
+### Workflow commands
```

Comment:
I'm not sure how useful this is - rare to need to do any of these in the terminal.

Something on troubleshooting would be more useful - I think merge conflicts, reverting changes (unstaged - staged - committed - pushed)

Reply:
I added some stuff here, but I need you to check, I don't think it's complete.